packages/flutter/lib/src/widgets/debug.dart

```diff
@@ -7,6 +7,7 @@ import 'dart:developer' show Timeline; // to disambiguate reference in dartdocs
 
 import 'package:flutter/foundation.dart';
 
+import 'basic.dart';
```

Comment:
This creates an import cycle because `basic.dart` imports this file.  Maybe that's fine.  Dart can cope with import cycles just fine.

Reply:
The import cycle already existed because this imports things like table.dart which import basic.dart.

Reply:
(But yeah I had the same thought. It seems unclean. But then it's just a file of debugging hooks, so I'm not too worried about this being the sign of a bad design. Debugging hooks always want to have their hands in everything, that's how we get good error messages...)